pmlc/conversion/stdx_to_spirv/stdx_to_spirv.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020, Intel Corporation
+
+#include "mlir/Dialect/SPIRV/SPIRVDialect.h"
+#include "mlir/Dialect/SPIRV/SPIRVOps.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#include "pmlc/conversion/stdx_to_spirv/pass_detail.h"
+#include "pmlc/conversion/stdx_to_spirv/passes.h"
+#include "pmlc/dialect/stdx/ir/ops.h"
+#include "pmlc/util/logging.h"
+
+using namespace mlir; // NOLINT[build/namespaces]
+
+namespace pmlc::conversion::stdx_to_spirv {
+
+namespace stdx = dialect::stdx;
+
+namespace {
+struct StdxUitoFOpConversion : public OpConversionPattern<stdx::UIToFPOp> {
+  using OpConversionPattern<stdx::UIToFPOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(stdx::UIToFPOp uitofOp, ArrayRef<Value> operands,
```

Comment:
Which test case needs this convertion?

Reply:
All the tests prefixed with testSparseCategoricalCrossentropy* use stdx.uitofp. Interesting, I checked again and found there's no stdx::UIToFPOp and std::UIToFPOp is used instead. May I ask when did this happen?

Reply:
stdx is plaidml extension of std dialect, when mlir std dialect get updated, some ops of the extension will be replaced by corresponding std ops.

Reply:
Yes I just realized this. Then this whole conversion can be removed. A conversion from std::UIToFPOp to spirv::ConvertUToFOp is need. It should be much similar like @XinWangIntel 's patch for converting [SIToFPOp](https://github.com/Flex-plaidml-team/plaidml/commit/f33a226107734ce81d56ca42e75135ab38dd869f).

Reply:
By the way, those tests mentioned above are stilling failing because their operands are i1. Any thoughts?

Reply:
It seems i1 could appear in quite a few ops. We have to think about the whole strategy processing i1. Liyang has some related work as well.